alt/content.md

```diff
@@ -0,0 +1,15 @@
+# ALT Linux
+
+This image serves as the `official ALT image` for the [ALT Linux](https://altlinux.org/).
+
+%%LOGO%%
+
+# About this image
+
+The `%%IMAGE%%:latest` tag will always point the latest stable release (which is, at the time of this writing, `%%IMAGE%%:p8`).
+
+## How It's Made
+
+The rootfs tarballs for this image are built using the [mkimage](https://en.altlinux.org/Mkimage) and [hasher](https://en.altlinux.org/Hasher), with an explicit goal being that they are transparent and reproducible. Using the same toolchain, it should be possible to regenerate (clean-room!) the same tarballs used for building the official Debian images.
+
+Additionally, the scripts in [%%GITHUB-REPO%%](%%GITHUB-REPO%%) are used to create each tag's `Dockerfile` and collect architecture-specific tarballs into a single place (for placement into [`dist-ARCH` branches on the same repository](%%GITHUB-REPO%%/branches), which also contain extra metadata about the artifacts included in each build, such as explicit package versions).
```

Comment:
This bit also appears to be cargo-culted from the Debian docs and doesn't appear to be applicable to ALT's repo.